gnosis/safe-cmm-app-react
# Feature/#7/total brackets component

# Update Aug 25th

- Input text now centered \o/
- Component complete
- Both Chrome and FF
- Lots of noise when refactoring component folders

![screenshot_2020-08-25_12-40-16](https://user-images.githubusercontent.com/43217/91223191-1ef4be80-e6d5-11ea-9d2f-aea74d4bf504.png) ![screenshot_2020-08-25_12-40-06](https://user-images.githubusercontent.com/43217/91223196-1f8d5500-e6d5-11ea-97bd-a28720794082.png)


# Description

Add total brackets component defined on [the design](https://projects.invisionapp.com/d/main/default/#/console/20213446/427462294/inspect)
![screenshot_2020-08-24_13-11-01](https://user-images.githubusercontent.com/43217/91091320-441cfa80-e60b-11ea-912c-abf858349655.png)

Current:
![screenshot_2020-08-24_13-11-32](https://user-images.githubusercontent.com/43217/91091374-55fe9d80-e60b-11ea-900c-fd8b7309ee07.png)
Struggling to get the input to center vertically.


# To Test
1. Checkout project, install dependencies with npm: `npm i`
2. Run storybook: `npm run storybook`
3. Visit http://localhost:6006/?path=/docs/bracketsinput--filled

- [ ] Stories should match design

# Background
